be/src/olap/compaction.cpp

```diff
@@ -339,6 +340,28 @@ Status Compaction::do_compaction_impl(int64_t permits) {
                      << ", output_version=" << _output_version;
         return Status::Error<ROWSET_BUILDER_INIT>();
     }
+    // now we support delete in cumu compaction, to make all data in rowsets whose version
```

Comment:
Capitalizes the first letter
```suggestion
    // Now ...
```